frappe/patches/v12_0/remove_deprecated_fields_from_doctype.py

```diff
@@ -1,10 +1,12 @@
 import frappe
 
 def execute():
-	frappe.reload_doc('core', 'doctype', 'doctype')
-	frappe.model.delete_fields({
```

Comment:
@josejibin Can you fix this method instead of replacing it with raw query...

I guess that method ([frappe.model.delete_fields](https://github.com/frappe/frappe/blob/develop/frappe/model/__init__.py#L70)) only needs quote replacement for a fix.